src/js/consts.js

```diff
@@ -112,6 +115,7 @@ export const visibilityFunctions = {
       return false;
     }
   },
+  featureFlag: checkFeatureFLag,
```

Comment:
Any reason not to define it here, but rather on line 38?

Reply:
I just don't like anonymous functions that's it.

Gotta figure out the test though